components/LocationInput.tsx

```
@@ -197,14 +276,24 @@ export default function LocationInput({
       )}
 
       {/* Display selected location info */}
-      {selectedLocation && (
+      {selectedLocation && selectedLocation.isCompleteAddress && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)